File: leetCode1822.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        prod_ = 1
        for i in range(len(nums)):
            prod_ *= nums[i]
        #print(prod_)

        if(prod_ > 0):
            return 1
        elif(prod_ == 0):
            return 0
        return -1

File: leetCode1822.java
class Solution {
    public int arraySign(int[] nums) {
        int product = 1;
        for(int i = 0; i < nums.length; i++) {
            if(nums[i] > 0)
                product *= 1;
            else if(nums[i] < 0)
                product *= -1;
            else product *= 0;
        }
        System.out.println(product);
        
        if(product > 0)
            return 1;
        else if(product == 0)
            return 0;
        else return -1;
    }
}